app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ProjectsHelper
+    def show_project_snippet?(project, user)
+      CollectionBlock.find_by(collection_id: project.id, user_id: user.id).nil?
```

Comment:
This should already have a `@user` and `@collection` variable.

Reply:
Here the `@user` and `@collection` variable can't be use and is not feasible if you review the  view `fromthepage/app/views/dashboard/landing_page.html.slim `
from where this helper method is being used you will get the idea.